relay-event-normalization/src/normalize/ai_model_costs.rs

```diff
@@ -0,0 +1,176 @@
+//! This file maps AI model IDs to their respective costs.
+//! Adapted from <https://github.com/langchain-ai/langchain/blob/5b6d1a907df8a7af0f47c5ffd394b3673d9539e5/libs/community/langchain_community/callbacks/openai_info.py#L4>
+
+/// Cost is in US dollars per thousand tokens
+fn calculate_cost_per_1k_tokens(model_id: &str, for_completion: bool) -> Option<f64> {
```

Comment:
What does "for_completion" mean? Is it input/output?

Reply:
Some models charge more for "completion tokens" than "prompt tokens", i.e., the model responding is more expensive than asking a question.